app/views/people/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class='name-container'>
+Name: <%= @people.name %> <br/><br/>
+Hobby: <%= @people.hobby %><br/><br/>
+Catchphrase: <%= @people.catchphrase %><br/><br/>
+</div>
+<br/><br/>
+<div class='name-container'>
+Tasks Not Complete: <%= Task.where(person_id: @people.id, task_completion: nil).length %><br/><br/>
+Tasks Completed: <%= Task.where(person_id: @people.id).where.not(task_completion: nil).length %><br/><br/>
```

Comment:
It's hard sometimes, but try to keep as much Model logic out of the Views as possible! It makes for better code and can make debugging/testing things much easier!